### PR TITLE
vk/Orlp-260/Access by URL

### PR DESCRIPTION
**Pull request by KuryloVolodymyr**
Now unauthorised user can`t access by url pages that are available to authorised user.

### CODE REVIEW

File: src/app/page/userProfile/profile.component.html [hunk 1]
```diff
@@ -105,6 +105,9 @@ <h5 *ngIf="userProfile" class="title">You are signed in as {{userProfile.email}}
           <button [disabled]="(firstName=='')||(lastName=='')" (click)="saveChanges()" type="button"
                   class="btn btn-outline-info">Save changes
           </button>
+          <button (click)="cancelChanges()" type="button"
```
Comment (antonbakalets) [inline src/app/page/userProfile/profile.component.html]: These are the changes from a different pull request. They should not be here.

File: .angular-cli.json [hunk 2]
```diff
@@ -21,7 +21,7 @@
       "styles": [
         "assets/css/myStyles.css",
         "scss/style.scss",
-        "../node_modules/font-awesome/css/font-awesome.css",
```
Comment (antonbakalets) [inline .angular-cli.json]: Why this change?

File: src/app/page/cabinet/cabinet.component.ts [hunk 3]
```diff
@@ -33,17 +34,23 @@ export class CabinetComponent implements OnInit {
...
               private router: Router,
               private userStatusChangeService: UserStatusChangeService,
-              private logger: NGXLogger) {
+              private logger: NGXLogger,
+              private logoutService: LogoutService) {
 
   }
 
   ngOnInit(): void {
+    this.isAuthorized = this.logoutService.isAuthorized();
```
Comment (antonbakalets) [inline src/app/page/cabinet/cabinet.component.ts]: You don't need the private field isAuthorized as the value is used only once after initialization and not reused.

Comment (KuryloVolodymyr) [reply src/app/page/cabinet/cabinet.component.ts]: So you recomend to change 
 `if (!this.isAuthorized) `
to 
`if(!this.logoutService.isAuthorized();)`
?

Comment (antonbakalets) [reply src/app/page/cabinet/cabinet.component.ts]: Yes, but you can disagree.
